File: android/src/main/java/com/kiwi/fluttercrashlytics/CrashActivity.java
package com.kiwi.fluttercrashlytics;

import android.app.Activity;
import android.os.Bundle;
import io.fabric.sdk.android.Fabric;
import com.crashlytics.android.Crashlytics;

public class CrashActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
        super.onCreate(savedInstanceState);
        Fabric.with(this, new Crashlytics());

        throw (FlutterException) getIntent().getSerializableExtra("exception");

    }
}

File: android/src/main/kotlin/com/kiwi/fluttercrashlytics/CrashActivity.kt
package com.kiwi.fluttercrashlytics

import android.app.Activity
import android.os.Bundle
import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.Fabric

class CrashActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
        super.onCreate(savedInstanceState)
        Fabric.with(this, Crashlytics())

        throw intent.getSerializableExtra("exception") as Throwable
    }
}
